src/xdocs/config_filters.xml

```diff
@@ -110,11 +110,8 @@
             file is edited.
           </p>
           <p>
-            Usage: This filter only works in conjunction with a <code>FileContentsHolder</code>, since that check makes
```

Comment:
@romani 
Should we add a note that FileContentsHolder is not required any more to prepare users for next step: removal of FileContentsHolder ?